expat/meta.yaml

```diff
@@ -0,0 +1,20 @@
+package:
+  name: expat
+  version: 2.1.0
+
+source:
+  fn: expat-2.1.0.tar.gz
+  url: http://sourceforge.net/projects/expat/files/expat/2.1.0/expat-2.1.0.tar.gz
+  md5: dd7dab7a5fea97d2a6a43f511449b7cd
+
+requirements:
+  build:
+    - gcc
```

Comment:
I don't feel comfortable about not adding it. It has been our custom to always add `libgcc` when we demand `gcc`. Or not use `gcc` at all (as Mike suggested above).